### DISCUSSION TITLE
Any more backend support instred of Rust?

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [tauri-apps/tauri#10409] [Q&A] [answered]
Any more backend support instead of Rust? Rust is too difficult to use.

**Comment by FabianLars** [comment, marked as answer]
Not yet, the feature is tracked here https://github.com/tauri-apps/tauri/issues/4670 - until then you can use the [sidecar](https://tauri.app/v1/guides/building/sidecar) feature to implement parts of your app in the language of your choice.
